dataland-frontend/src/components/resources/frameworkDataSearch/TwoLayerDataTable.vue

```diff
@@ -212,7 +231,7 @@ export default defineComponent({
      * @param modalTitle The title for the modal, which is derived from the key of the KPI
      * @param kpiKey the key of the KPI used to determine the type of Subtable that needs to be displayed
      */
-    openModalAndDisplayValuesInSubTable(listOfValues: [], modalTitle: string, kpiKey: string) {
+    openModalAndDisplayValuesInSubTable(listOfValues: string[], modalTitle: string, kpiKey: string) {
```

Comment:
I think this change is wrong.   The method actually receives an array of objects.

Reply:
(See its usage in the DOM)

Reply:
changed: string[] to []

Reply:
or should we change it to object[]  ?

Reply:
vote for  [] :)

Reply:
👍  closing